src/OrchardCore/OrchardCore.DisplayManagement.Liquid/Filters/LiquidViewFilters.cs

```diff
@@ -84,11 +88,57 @@ public static Task<FluidValue> ClearAlternates(FluidValue input, FilterArguments
             return Task.FromResult(input);
         }
 
+        public static Task<FluidValue> AddAlternates(FluidValue input, FilterArguments arguments, TemplateContext context)
+        {
+            if (input.ToObjectValue() is Shape shape)
+            {
+                var alternates = arguments["alternates"].Or(arguments.At(0));
+
+                if (alternates.Type == FluidValues.Array)
+                {
+                    foreach (var value in alternates.Enumerate())
+                    {
+                        shape.Metadata.Alternates.Add(value.ToStringValue());
+                    }
+                }
+            }
+
+            return Task.FromResult(input);
+        }
+
+        public static Task<FluidValue> ClearClasses(FluidValue input, FilterArguments arguments, TemplateContext context)
+        {
+            if (input.ToObjectValue() is Shape shape && shape.Classes.Count > 0)
+            {
+                shape.Classes.Clear();
+            }
+
+            return Task.FromResult(input);
+        }
+
+        public static Task<FluidValue> AddClasses(FluidValue input, FilterArguments arguments, TemplateContext context)
+        {
+            if (input.ToObjectValue() is Shape shape)
+            {
+                var classes = arguments["classes"].Or(arguments.At(0));
+
+                if (classes.Type == FluidValues.Array)
```

Comment:
Handle also when it's a string, and split it by yourself

Reply:
Ok, i was too lazy.